tests/pytorch/test_nn.py

```diff
@@ -1801,3 +1801,22 @@ def test_BiasedMultiheadAttention(feat_size, num_heads, bias, attn_bias_type, at
     out = net(ndata, attn_bias, attn_mask)
 
     assert out.shape == (16, 100, feat_size)
+
+@pytest.mark.parametrize('max_len', [1, 4])
+@pytest.mark.parametrize('feat_dim', [8, 16])
+@pytest.mark.parametrize('num_heads', [1, 8])
+def test_PathEncoder(max_len, feat_dim, num_heads):
```

Comment:
Extend the test to cover GPU with `F.ctx()`.

Reply:
OK